tests/tests.rs

```diff
@@ -1911,3 +1911,55 @@ fn serde() {
         "invalid value: integer `-1`, expected u128",
     );
 }
+
+#[cfg(all(feature = "borsh", feature = "std"))]
+#[test]
+fn borsh() {
+    use borsh::schema::BorshSchemaContainer;
+    use borsh::{BorshDeserialize, BorshSerialize};
+    let mut buf = Vec::new();
+    let input = u9::new(42);
+    input.serialize(&mut buf).unwrap();
+    let output = u9::deserialize(&mut buf.as_ref()).unwrap();
+    assert_eq!(buf.len(), (u9::BITS + 7) / 8);
+    assert_eq!(input, output);
+
+    let input = u63::MAX;
+    let mut buf = Vec::new();
+    input.serialize(&mut buf).unwrap();
+    let output: u63 = u63::deserialize(&mut buf.as_ref()).unwrap();
+    assert_eq!(buf.len(), (u63::BITS + 7) / 8);
+    assert_eq!(input, output);
+
+    let schema = BorshSchemaContainer::for_type::<u9>();
```

Comment:
just to understand this: BorschSchemaContainer does NOT require schemars? So we have two different things that are called schema?

Reply:
One schema is JSON/OpenAPI shema integrated with serde.

Other shema is borshschema integrated with borsh.

There are schemas for protobuf/scale/etc.

Reply:
do not require, just supporting several schemas